website/docs/r/spot_fleet_request.html.markdown

```diff
@@ -142,3 +142,11 @@ In addition to all arguments above, the following attributes are exported:
 
 * `id` - The Spot fleet request ID
 * `spot_request_state` - The state of the Spot fleet request.
+
+## Import
+
+Spot Fleet Requests can be imported using `id`, e.g.
+
+```
+$ terraform import aw_spot_fleet_request.fleet sfr-005e9ec8-5546-4c31-b317-31a62325411e
```

Comment:
```suggestion
$ terraform import aws_spot_fleet_request.fleet sfr-005e9ec8-5546-4c31-b317-31a62325411e
```

Reply:
I didn't catch this on first review, small typo @DrFaust92

Reply:
fixed

Reply:
nice, thank you!